tests/SprykerEcoTest/Yves/Computop/_support/Mapper/PayPalExpressToQuoteMapperTestHelper.php

```diff
@@ -0,0 +1,89 @@
+<?php
+
+namespace SprykerEcoTest\Yves\Computop\Mapper;
+
+use Codeception\TestCase\Test;
+use Generated\Shared\Transfer\ComputopPayPalExpressInitResponseTransfer;
+use Generated\Shared\Transfer\ComputopPayPalExpressPaymentTransfer;
+use Generated\Shared\Transfer\ExpenseTransfer;
+use Generated\Shared\Transfer\ItemTransfer;
+use Generated\Shared\Transfer\PaymentTransfer;
+use Generated\Shared\Transfer\QuoteTransfer;
+use Generated\Shared\Transfer\ShipmentTransfer;
+use Generated\Shared\Transfer\TotalsTransfer;
+
+class PayPalExpressToQuoteMapperTestHelper extends Test
```

Comment:
Move all the logic from this class to `ComputopYvesTester`. It is exactly what the tester is designed for.